src/applications/personalization/profile-2/tests/e2e/profile.cypress.spec.js

```diff
@@ -0,0 +1,105 @@
+import { PROFILE_PATHS } from '../../constants';
+
+import mockUser from '../fixtures/users/user-36.json';
+import mockPaymentInfo from '../fixtures/payment-information/direct-deposit-is-set-up.json';
+import mockFeatureToggles from '../fixtures/feature-toggles.json';
+
+/**
+ *
+ * @param {boolean} mobile - test on a mobile viewport or not
+ *
+ * This helper:
+ * - loads the Profile, confirms the user is redirected to the correct URL, and
+ *   then performs an aXe scan
+ * - clicks through each item in the sub-nav, checks that the URL is correct,
+ *   and performs an aXe scan
+ */
+function checkAllPages(mobile = false) {
+  cy.visit('/profile');
+  if (mobile) {
+    cy.viewport('iphone-6');
```

Comment:
Do we always want to default to iPhone 6? Shouldn't we also run the tests on Android? I am not too familiar with how `vets-website` deals with mobile operating systems in Nightwatch currently

Reply:
As per our chat I'll switch this to an even smaller screen size.